tests/bluetooth/audio/mocks/include/mock_kernel.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef MOCKS_KERNEL_H_
+#define MOCKS_KERNEL_H_
+
+#include <zephyr/fff.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys_clock.h>
```

Comment:
What is included from `sys_clock.h` here?

Reply:
k_ticks_t